consumers/hidden-test-consumer/src/hidden_test_consumer/process.py

```
@@ -0,0 +1,298 @@
+        try:
+            with zipfile.ZipFile(
+                f"{self.download_dir}/hidden-tests.zip", "r"
+            ) as zip_ref:
+                zip_ref.extractall(f"{self.download_dir}/extracted")
+        except Exception as e:
+            yield ProcessRequest(
+                status=Status.ERROR, message="❌ Error unzipping hidden test data"
+            )
+            raise e
+
+    def check_directory_structure(self) -> Generator[ProcessRequest, None, None]:
```

Ruff (0.8.2)

`check_directory_structure` is too complex (12 > 10)

(C901)